shared/featureconfig/config.go

```diff
@@ -331,35 +342,40 @@ func ConfigureSlasher(ctx *cli.Context) {
 // ConfigureValidator sets the global config based
 // on what flags are enabled for the validator client.
 func ConfigureValidator(ctx *cli.Context) {
-	// Using Medalla as the default configuration for now.
-	params.UseMedallaConfig()
-
 	complainOnDeprecatedFlags(ctx)
 	cfg := &Flags{}
 	if ctx.Bool(AltonaTestnet.Name) {
```

Comment:
This testnet bootstrapping seems to be duplicated for each binary configuration. Can you refactor to something like 
`configureTestnet(ctx, cfg)`?